react-native-scripts/src/scripts/init.js

```diff
@@ -71,6 +86,8 @@ https://github.com/npm/npm/issues/16991
     eject: 'react-native-scripts eject',
     android: 'react-native-scripts android',
     ios: 'react-native-scripts ios',
+    web: './node_modules/.bin/webpack-dev-server -d --config ./webpack.config.js  --inline --hot --colors --content-base public/',
```

Comment:
no need to scope with `./node_modules` here, we can just do `webpack-dev-server`